keyboards/keebio/dsp40/rev1/rules.mk

```
@@ -12,7 +12,7 @@ MOUSEKEY_ENABLE = yes       # Mouse keys
 EXTRAKEY_ENABLE = yes       # Audio control and System control
 CONSOLE_ENABLE = yes        # Console for debug
 COMMAND_ENABLE = yes        # Commands for debug and configuration
-NKRO_ENABLE = no            # Enable N-Key Rollover
+NKRO_ENABLE = yes            # Enable N-Key Rollover
```

```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```
Modifications at the keyboard level that are inconsistent with QMK upstream will cause conflicts later down the road. If you want this change, [submit a PR to QMK](https://github.com/qmk/qmk_firmware/pulls). Otherwise, confine it to the Vial keymap's `rules.mk` file.